pennylane/_grad.py

```diff
@@ -27,11 +27,10 @@
 
 
 class grad:
-    """A :func:`~.qjit` compatible gradient transformation that returns the gradient
-    as a callable function of (functions of) QNodes.
+    """Returns the gradient as a callable function of (functions of) QNodes.
```

Comment:
@josh146 I removed the qjit support from the headline. We link to `catalyst.grad` and `catalyst.jacobian` doc pages for more information in these doc pages as well as there's now a new list of QJIT compatible PL ops in the compiler doc page. I can revert this if you think otherwise. cc:  @timmysilv